hs_core/templates/email/group_membership.html

```diff
@@ -0,0 +1,14 @@
+{% load i18n %}
+{% block main %}
+    {% if group_owner %}
+        <p>{% trans "Dear " %}{{ group_owner.first_name }}</p>
+        <p>{% trans "User " %}{{ user.first_name }} has requested to join your HydroShare group ({{ group.name }}). Please click on the link below to accept this request.</p>
```

Comment:
@hyi has a good idea here.  If I'm a group owner and I get an email stating that a HydroShare user called "Stinky_Pete_133" wants to join my HydroShare group, I have no idea who that person is.  So, you should either include additional information in the email about who the user is, or provide a link to that user's HydroShare profile page so the group owner could check them out first.